site/pages/docs/global/spacings.vue

```diff
@@ -16,8 +16,8 @@
         <p><o-label text="SCSS" size="small" /> recall spacings with <code>margin: map-get($spaces, {key});</code>, <code>padding: map-get($spaces, {key});</code>, etc.</p>
         <p><o-label text="HTML" size="small" /> apply margin to elements with <code>.m-{key}</code>, <code>.ml-{key}</code>, <code>.mr-{key}</code>, <code>.mt-{key}</code>, <code>.mb-{key}</code> classes</p>
         <p><o-label text="HTML" size="small" /> apply padding to elements with <code>.p-{key}</code>, <code>.pl-{key}</code>, <code>.pr-{key}</code>, <code>.pt-{key}</code>, <code>.pb-{key}</code> classes</p>
-        <p><o-label text="Responsive" size="small" />  apply responsive margin to elements <code>.{breakpoint}-m-{key}</code>, <code>.{breakpoint}-ml-{key}</code>, <code>.{breakpoint}-mr-{key}</code>, <code>.{breakpoint}-mt-{key}</code>, <code>.{breakpoint}-mb-{key}</code> classes</p>
-        <p><o-label text="Responsive" size="small" />  apply responsive padding to elements <code>.{breakpoint}-p-{key}</code>, <code>.{breakpoint}-pl-{key}</code>, <code>.{breakpoint}-pr-{key}</code>, <code>.{breakpoint}-pt-{key}</code>, <code>.{breakpoint}-pb-{key}</code> classes</p>
+        <p><o-label text="Responsive" size="small" />  apply responsive margin to elements <code>.{breakpoint}:m-{key}</code>, <code>.{breakpoint}:ml-{key}</code>, <code>.{breakpoint}:mr-{key}</code>, <code>.{breakpoint}:mt-{key}</code>, <code>.{breakpoint}:mb-{key}</code> classes</p>
+        <p><o-label text="Responsive" size="small" />  apply responsive padding to elements <code>.{breakpoint}:p-{key}</code>, <code>.{breakpoint}:pl-{key}</code>, <code>.{breakpoint}:pr-{key}</code>, <code>.{breakpoint}:pt-{key}</code>, <code>.{breakpoint}:pb-{key}</code> classes</p>
```

Comment:
Looking at the `Responsive` tag docs, i would move it into docs section with Heading and not mix it up with `SCSS/HTML` basic docs.

![Screenshot 2019-10-07 at 06 38 14](https://user-images.githubusercontent.com/16240817/66285583-b2f74500-e8cd-11e9-9710-c78ff1a6adb7.png)

Reply:
ye i agree, i just wasn't sure about this what to do... will try something now

Reply:
Check it out, i have added changes to others also `text-align`, `font sizes` etc.